includes/gateways/manual.php

```diff
@@ -32,7 +31,7 @@
  * @return bool
  **/
 function give_manual_form_ouput() {
-	if ( FormTemplateUtils::getActiveId( FrontendFormTemplateUtils::getFormId() ) === 'legacy' ) {
+	if ( FormUtils::isLegacyForm( $form_id ) ) {
```

Comment:
@henryholtgeerts This will throw PHP notice because `$form_id` is not defined. You can remove this.

Reply:
fixed: https://github.com/impress-org/givewp/compare/66114f9b23fd...c5481037f8ec